droughtpipeline/load.py

```diff
@@ -417,11 +406,71 @@ def send_to_ibf_api(
                             self.ibf_api_post_request("admin-area-dynamic-data/exposure", body=body )
                     processed_pcodes = list(set(processed_pcodes))
 
+                elif lead_time_event == 0:
+                    season_name = expected_events[lead_time_event]
+                    if climate_region_name.split('_')[0] in ['National','national']:
+                        event_name = season_name 
+                    else:
+                        event_name = f"{climate_region_name} {season_name}"
+                    datestart, dateend = self.__look_up_time(
+                        country, 
+                        event_name, 
+                        lead_time_event='1-month',)
+                    forecast_data = self.get_pipeline_data(
+                                    'seasonal-rainfall-forecast', 
+                                    country=country,
+                                    start_date=datestart,
+                                    end_date=dateend,
+                    )
```

Comment:
so I understand that you just get a start and end date from the cosmos-db and use those to do a normal data-ingestion, just like you would in normal pipeline runs, right? 

This would be instead of directly getting the exact exposure data from cosmos-db, and upload that again, only with different leadTime and date. (which was my interpretation).

Anyway, I guess your way works too. Are we certain that historical data we get from that source does not change any more afterwards (which would be a prerequisite for this method).